README.md

```diff
@@ -1,6 +1,48 @@
 # PSR - Log Util
 Utilities for the [`psr/log`][] package that backs the [PSR-3][] specification.
 
+ * `Psr\Log\Test\LoggerInterfaceTest` provides a base test class for ensuring compliance with the LoggerInterface.
+ * `Psr\Log\Test\TestLogger` is a mock class for testing purposes.
+
+## Installation
+
+This package should be used only for tests.
+
+For PHP 8.0+, you may remove support for older versions of `psr/log`:
```

Comment:
I'm not sure this is a good suggestion. Leaving compatibility for older versions makes the package backward compatible, so I would leave at least `^2.0`